README.md

```diff
@@ -96,7 +96,55 @@ java -jar build/libs/dev-sourcecode-record-0.0.2-SNAPSHOT-all.jar convert-to-git
 
 ## The SRCS file format
 
-TODO 
+The SRCS file format is divided into two parts.
+
+1. Header
+
+    The header contains 14 bytes: 6 for magic bytes `SRCSTM` and 8 byte to store
+    UNIX timestamp when the snapshot was first recorded. The timestamp is stored
+    in Little Endian format.
+
+2. Segments
+
+    After the header, the body will contains several segments that contains
+    snapshot of the working directory. The content of the segments are
```

Comment:
It would be nice to have a Markdown table to summarise the fields.
https://help.github.com/articles/organizing-information-with-tables/

Reply:
Yes, it looks much better now. Thanks for the idea!

Reply:
Nice!